package.json

```diff
@@ -2,54 +2,51 @@
   "name": "@adobe/asset-compute-sdk",
   "description": "Adobe Asset Compute Worker SDK",
   "license": "Apache-2.0",
-  "version": "27.0.0",
+  "version": "1.0.0",
   "author": {
     "name": "Adobe Inc."
   },
-  "publishConfig": {
-    "registry": "https://artifactory.corp.adobe.com/artifactory/api/npm/npm-nui-release-local"
-  },
   "repository": {
     "type": "git",
-    "url": "https://git.corp.adobe.com/nui/asset-compute-sdk.git"
+    "url": "adobe/asset-compute-sdk"
```

Comment:
Instead of:

```
"repository": {
     "type": "git",
     "url": "adobe/asset-compute-sdk"
}
```

just
```
"repository": "adobe/asset-compute-sdk"
```

See https://docs.npmjs.com/files/package.json#repository

Reply:
(changed)